pkg/kernel/networkservice/inject/server.go

```diff
@@ -45,32 +45,30 @@ func (s *injectServer) Request(ctx context.Context, request *networkservice.Netw
 		return next.Server(ctx).Request(ctx, request)
 	}
 
-	if err := move(logger, request.GetConnection(), false); err != nil {
+	if err := move(ctx, request.GetConnection(), false); err != nil {
 		return nil, err
 	}
 
 	conn, err := next.Server(ctx).Request(ctx, request)
 	if err != nil {
-		err = move(logger, request.GetConnection(), true)
-		if err != nil {
-			logger.Warnf("server request failed, failed to move back the interface: %v", err)
+		moveRenameErr := move(ctx, request.GetConnection(), true)
```

Comment:
Please use here `isEstablished` check as we discussed offline.

Reply:
done